webpack.dev.config.js

```diff
@@ -60,6 +60,8 @@ module.exports = {
             loader: 'css-loader',
             options: {
               sourceMap: true,
+              importLoaders: 2,
```

Comment:
What does this do?

Reply:
@andre-meyer https://github.com/webpack-contrib/css-loader/issues/228